ivi-layermanagement-api/ilmCommon/include/ilm_types.h

```diff
@@ -287,4 +287,33 @@ typedef void(*notificationFunc)(ilmObjectType object,
 typedef void(*shutdownNotificationFunc)(t_ilm_shutdown_error_type error_type,
                                         int errornum,
                                         void* user_data);
+
+/**
+ * Typedef for notification callback on screenshot send done event
+ * @param user_data the use data, be passed when call the screenshot api
+ * @param fd fd for file containing image data, don't close it in callback,
+ * it will be closed after the callback execution
```

Comment:
please add the comment that the FD should not be accessed any longer after the callback execution

Reply:
I updated the description. Thanks.
@audoan99 I think this ready to merge. Could you please have check with this?